lib/pbench/server/database/alembic/versions/62eddcec4817_add_dataset_access.py

```diff
@@ -0,0 +1,45 @@
+"""Add Dataset.access
```

Comment:
Why do we need alembic version file? We dont have any database to upgrade

Reply:
Webb commented on this earlier. Really, I started it because I was making a change to the database schema, which is the point of including Alembic, and I'd never tried to use it. In the process, I found some stale and buggy and confusing code that I was able to clean up, and that's all good. I left this particular update file as, more than anything else, a concrete example of how to use it as well as a demonstration that it actually works. I updated the live schema on my test VM using alembic rather than doing it manually in `psql` or deleting the entire DB and starting from scratch, and that was a useful experiment.

In the long run, I agree we possibly want to generate Alembic version files only for "actual" releases, although that still means that every development deployment might have to start by deleting the existing DB. The alternate philosophy would be that every DB schema change comes with an Alembic version so that we can always update any deployed server (whether production or development) to the current schema without disrupting any existing data.

The later is convenient for development, although it means carrying along a lot more version files and probably slightly slower overall DB upgrades during deployment. Part of the tradeoff relies on assumptions of how often (and how radically) we'll be changing the schema. Personally I like the idea that any deployed server can be upgraded dynamically to the latest schema, and I'm in favor of an Alembic version for each PR that changes the schema rather than collecting them and trying to get all the updates right at the end of a release in a single file.